packages/cli/src/api/utils.ts

```diff
@@ -9,6 +12,15 @@ export function prettyOrigin(origins: [filename: string, line?: number][]) {
   }
 }
 
+export function replacePlaceholders(
+  input: string,
+  values: Record<string, string>
+): string {
+  return input.replace(/\{([^}]+)}/g, (m, placeholder) => {
```

Comment:
When I provide the following input:
`/foo/bar/{a{b}`
and the following values:
`{"b": "test"}`

this regex won't replace the `{b}` in the input with the string `test` as the matched part of the string is `a{b` instead of `b`.
You are testing just happy paths in the test file `utils.test.ts`.

Reply:
this is obviously a configuration mistake, i'm not going to handle this case.

Reply:
it's not going to be a full-powered template engine, as long as what we do is not a templates. We just need to replace one placeholders to value.